volontulo/tests/common.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+u"""
+.. module:: common
+"""
+
+from django.contrib.auth.models import User
+
+from volontulo.models import Badge
+from volontulo.models import Offer
+from volontulo.models import Organization
+from volontulo.models import UserBadges
+from volontulo.models import UserProfile
+
+
+class Common(object):
+    u"""Handle helper methods used in many places."""
+
+    @classmethod
```

Comment:
There is no necessary to define this functions as classmethods. We can easly use staticmethods.